pom.xml

```diff
@@ -1,28 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
make sure with Amit pls its working in here project after the PR- she wrote it.

Reply:
It just fixes a warning, specifies the encoding for different systems. It should work, but we should try it on other platforms.